docs/dev/framework/asset-management.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Asset Management"
+description: Managing assets (JavaSCript, CSS) in the front end.
+---
+
+
+When building a site, administrator and developers can add CSS and JavaScript assets
+to their site via Contao's page layouts. However, as an application or extension
+developer, you might want to add your own assets to a page, whenever necessary.
+
+
+## Gobal Arrays
+
+Contao utilises global arrays to store references for later inclusion into the page
+layout.
+
+
+| Array | Description |
+| --- | --- |
+| `$GLOBALS['TL_BODY']` | Contains HTML code to be included before `</body>`.<sup>1</sup> |
+| `$GLOBALS['TL_CSS']` | Contains relative paths to CSS assets to be included in the `<head>` of the document. |
+| `$GLOBALS['TL_FRAMEWORK_CSS']` | Contains relative paths to CSS assets to be included in the `<head>` of the document. |
+| `$GLOBALS['TL_HEAD']` | Contains HTML code to be included in the `<head>` of the document. |
+| `$GLOBALS['TL_JAVASCRIPT']` | Contains relative paths to JavaScripts assets to be included in the `<head>` of the document. |
+| `$GLOBALS['TL_JQUERY']` | Contains HTML code to be included before `</body>`.<sup>1</sup> |
```

Comment:
jquery and mootools are only included if the respective checkbox is enabled in the layout, as far as I remember?

Reply:
No, that's always processed. Nothing will be _in_ those global arrays, if you haven't activated any templates for it - but if you add something to those arrays yourself, it will be shown in the front end.

Reply:
I have removed `TL_JQUERY` nonetheless.